platform-configs/raptor-cs_talos2.robot

```diff
@@ -105,6 +105,8 @@ ${RESET_TO_DEFAULTS_SUPPORT}=                       ${FALSE}
 ${MEMORY_PROFILE_SUPPORT}=                          ${FALSE}
 ${DEFAULT_POWER_STATE_AFTER_FAIL}=                  Powered Off
 ${ESP_SCANNING_SUPPORT}=                            ${FALSE}
+${L2_CACHE_SUPPORT}=                                ${TRUE}
```

Comment:
If the default becomes true, then it is not needed here.

Reply:
removed in: 2cde25bac3dd60533fbc565a0ca6010772a8d1b6 as it is no longer needed if the default is set to true